### DISCUSSION TITLE
Make MenuHelpCommand as hybrid

### DISCUSSION THREAD

**Discussion by DestructiveBeastro** [InterStella0/starlight-dpy#9] [Q&A] [answered]
I was wondering if I can make the starlight MenuHelpCommand a hybrid command so that the help command can be invoked as a slash command

**Comment by InterStella0** [comment, marked as answer]
This is already the case stated in the docs [here](https://starlight-dpy.readthedocs.io/en/latest/help-command.html#menuhelpcommand). It extends the `HelpHybridCommand`. You would need to set with_app_command to True to include the app command implementation on the `MenuHelpCommand`. By default it is set as False.

```python
bot.help_command = starlight.MenuHelpCommand(with_app_command=True)
```

After that just sync app command with your bot.
